Rujumba/Sacco-Frontend
# Ft loan

### **What does this PR do?**
This the user interface for application for a loan by the user

### **Task to be completed?**
To ensure that a user can be able to apply for a loan from the sacco
To ensure the admin can see the loan requests and verify them

### **How to manually test it?**
Fork this repository to your personal Github account
Clone the forked repo and navigate to it using the command cd Sacco-Frontend
Package the Sacco-Frontend using maven
Transfer the .war file into the server
Then run the server and it can be accessed port localhost:8080
After logging in, the dashboard is displayed with a navbar and then a user selects the apply for loan option.
A form for loan application is displayed
for the admin, this page shows the loan requests for verification

### **User side Screenshot**
![Screenshot (33)](https://github.com/Rujumba/Sacco-Frontend/assets/107847681/128b1279-e3fb-43a3-a8a7-a05d36fc6df4)

### **Admin side Screenshot**
![Screenshot (39)](https://github.com/Rujumba/Sacco-Frontend/assets/107847681/9ef3367e-449e-41b7-8b27-e5b4acf292c7)
